.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Where is the extra line at the end of this file? When you raised this PR did you go through the files and did you see this red marker?